src/util.test.ts

```diff
@@ -24,7 +24,8 @@ describe("Util", (): void => {
       command: string,
       error: Error | null,
       stdout: string = "",
-      stderr: string = ""
+      stderr: string = "",
+      platform: NodeJS.Platform
```

Comment:
There's no sense in having default arguments followed by an argument without a default in JavaScript since there's no way to use any of the defaults in that case. The default values can be removed if they aren't needed, and the order of the arguments can be switched if they are.